component/app.jsonnet

```diff
@@ -1,10 +1,10 @@
 local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
-local params = inv.parameters.grafana;
+local params = inv.parameters.grafana_helm;
 local argocd = import 'lib/argocd.libjsonnet';
 
-local app = argocd.App('grafana', params.namespace);
+local app = argocd.App('grafana_helm', params.namespace);
 
 {
-  grafana: app,
+  grafana_helm: app,
```

Comment:
```suggestion
  'grafana-helm': app,
```

Reply:
I get an error with this `STATIC ERROR: component/app.jsonnet:9:3-10: expected one of :, ::, :::, +:, +::, +:::, got: -` in `lint_jsonet`